corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -6,6 +6,25 @@
 
 {% requirejs_main 'users/js/mobile_workers' %}
 
+{% block page_navigation %}
```

Comment:
Should this include `{{ block.super }}`? [This](https://github.com/dimagi/commcare-hq/blob/72e7038a757626470614bc6fa5c6a8742f1df19b/corehq/apps/hqwebapp/templates/hqwebapp/two_column.html#L11) makes me think that without it, the sidebar won't show up.

Could you add a screenshot of what the notification looks like?

Reply:
<img width="1433" alt="Screen Shot 2020-09-28 at 2 14 52 PM" src="https://user-images.githubusercontent.com/36681924/94486100-e8ff9a00-01ac-11eb-8e16-a49a7d13d157.png">

Reply:
You are right-- it was originally blocking, but after adding {{ block.super }}, it appears below

Reply:
Cool, thanks. FYI @djmore9 in case you wanted to see how this will look.